README.md

```diff
@@ -10,18 +10,20 @@ This README will provide directions for building, testing, and debugging that co
 
 * NSM_NAME - A string value of network service client name (default "nsc")
 * NSM_CONNECT_TO - A Network service Manager connectTo URL (default "unix:///var/lib/networkservicemesh/nsm.io.sock")
-* NSM_MAX_TOKEN_LIFETIME -  A token lifetime duration (default 24h)                                                            
-* NSM_ROUTES - A comma separated list of routes.                                                                                 
+* NSM_MAX_TOKEN_LIFETIME -  A token lifetime duration (default 24h)
+* NSM_ROUTES - A comma separated list of routes.
 * NSM_LABELS - A list of client labels with format key1=val1,key2=val2, will be used a primary list for network services
-* NSM_MECHANISM - Default Mechanism to use, supported values "kernel","memif"                                                         
+* NSM_MECHANISM - Default Mechanism to use, supported values "kernel", "memif", "vfio"
 * NSM_NETWORK_SERVICES - A list of Network Service Requests URLs with inner format 
-    - [{mechanism}]?:${nsName}[@domainName]?/${interfaceName/memIfSocketName}?${label1}=${value1}&${label2}=${value2}
-        - mechanism one of "kernel", "memif"
+    - \[kernel://]nsName\[@domainName]\[:capability]/interfaceName?\[{labelN=valueN}]
+    - \[memif://]nsName\[@domainName]/memIfSocketName?\[{labelN=valueN}]
```

Comment:
Clients don't specify memIfSocketNames ... they are told them by the Connection.Mechanism.Parameters coming from the NSE.

Reply:
Again, clients don't specify memIfSocketName...

Reply:
OK, I guess we are not ready for memif client in here :)